app/utils/logger.js

```diff
@@ -0,0 +1,58 @@
+// app/utils/logger.js
+
+const { createLogger, transports, format } = require('winston');
+const { combine, timestamp, printf, colorize, splat, simple } = format;
+const fs = require('fs');
+const path = require('path');
+require('winston-daily-rotate-file');
+
+// Ensure log directory exists
+const logDirectory = path.join(__dirname, '../../logs');
```

Comment:
It's better to let the user select the log directory, for instance, set an item in the configuration file with the default value '../../log'. This will benefit when using different deployment methods (e.g.: if you are using docker, there might be a certain directory for log collection).